modules/microshift-default-settings.adoc

```diff
@@ -38,40 +38,42 @@ dns:
 etcd:
   memoryLimitMB: 0 # <5>
 ingress:
-  listenAddress:
-    - ""
+  listenAddress: <6>
+    - "" <7>
   ports:
     http: 80
     https: 443
```

Comment:
These are also customizable

Reply:
With a range or only single ports for each?